tests/queries/0_stateless/03224_modulo_or_null.sql

```diff
@@ -0,0 +1,7 @@
+select moduloOrNull(0, 0);  -- { serverError ILLEGAL_TYPE_OF_ARGUMENT }
+select moduloOrNull(1, materialize(0));  -- { serverError ILLEGAL_TYPE_OF_ARGUMENT }
```

Comment:
Why is that? Shouldn't `moduloOrNull(0, 0)` return `NULL`?

If we compare that with `moduloOrZero(0, 0)` which returns zero, then it seems we can expect that `moduloOrNull(0, 0)` will return `NULL`.

Reply:
> Why is that? Shouldn't `moduloOrNull(0, 0)` return `NULL`?
> 
> If we compare that with `moduloOrZero(0, 0)` which returns zero, then it seems we could expect that `moduloOrNull(0, 0)` would return `NULL`.

Yeah, u r right, we should also make this to NULL. Thanks for pointing it out.